### ISSUE TITLE
Cast time to UTC

### ISSUE THREAD

**Issue by butvinm** [MomsDeveloper/better-itmo-schedule-export#6]
Most Calendars ignore Timezone field and apply time offset to event bounds. So, it's better to export events in UTC

**Comment by worthant** [comment]
Don't care about quality honestly: (hotfix)

```python
from datetime import datetime, timedelta

def subtract_3_hours(time_str):
    dt = datetime.strptime(time_str, "%Y%m%dT%H%M%SZ")
    adjusted_dt = dt - timedelta(hours=3)
    return adjusted_dt.strftime("%Y%m%dT%H%M%SZ")

with open("calendar.ics", "r") as file:
    content = file.read()

lines = content.split("\n")
modified_lines = []

for line in lines:
    if line.startswith("DTSTART;VALUE=DATE-TIME:") or line.startswith("DTEND;VALUE=DATE-TIME:"):
        time_str = line.split(":")[1]
        adjusted_time_str = subtract_3_hours(time_str)
        modified_line = line.split(":")[0] + ":" + adjusted_time_str
        modified_lines.append(modified_line)
    else:
        modified_lines.append(line)

modified_content = "\n".join(modified_lines)

with open("modified_calendar.ics", "w") as file:
    file.write(modified_content)

print("Modified .ics file created: modified_calendar.ics")
```

**Comment by butvinm** [comment]
@worthant sorry, this issue already addressed in #7
